packages/relative-values/src/components/View/hooks/useRelativeValues.ts

```diff
@@ -12,7 +12,7 @@ const wrapper = sq`
     median: inv(dist, 0.5),
     min: inv(dist, 0.05),
     max: inv(dist, 0.95),
-    db: 10 * (dist -> log10 -> stdev)
+    db: 10 * (SampleSet.map(dist, abs) -> log10 -> stdev)
```

Comment:
For distributions that cross 0 this would sometimes fail, if one of the samples is too close to 0.

Should it be `SampleSet.map(dist, {|x| max([abs(x), 1e-9])})`?